app/views/decidim/consultations/admin/consultations/results.html.erb

```diff
@@ -7,6 +7,13 @@
         /
         <%= t "decidim.admin.consultations.results.participants", count: current_consultation.total_participants %>
       </span>
+      <span id="export-consultation-results" class="button--title">
+        <% if allowed_to?(:export_consultation_results, :consultation, { consultation: current_consultation }, [::Decidim::ActionDelegator::Permissions]) %>
```

Comment:
This is yet another example of what we need to solve in https://github.com/coopdevs/decidim-module-action_delegator/issues/24. Permissions are increasingly growing in complexity and that hampers changeability.